project/SaunaBuild.scala

```diff
@@ -32,7 +32,9 @@ object SaunaBuild extends Build {
         "com.typesafe.akka" %% "akka-testkit" % akkaVersion,
         "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
         "com.typesafe.akka" %% "akka-cluster-tools" % akkaVersion,
-
+        "org.json" % "json" % "20160212",
```

Comment:
I think we should try to avoid unnecessary dependencies, especially when something similar is here.
I guess, there shouldn't be absolutely anything in `org.json.json` that you cannot do with Play JSON.

Same for `commons-io`. I think if we can try to use Play WS (which awesome library by the way),
we can avoid this unnecessary dependency used only to decode output in UTF-8.